canonical/juju-dashboard
# WD-8541 - fix: Scroll to the top of Config Panel if error occurs

## Done

- Scroll to the top of Config Panel if error occurs.

#QA

- A possible way to QA the changes is to go to the config panel of an app that has a field which is required to be a number. Inspect the page, change the input type to `text`, write invalid data and then press "Save and apply". The panel should scroll to the top and the error should be displayed inline.

## Details

- https://warthogs.atlassian.net/browse/WD-8541
